app/helpers/people_helper.rb

```diff
@@ -32,7 +32,8 @@ def person_link(person, opts={})
   end
 
   def person_image_tag(person, size = :thumb_small)
-    image_tag(person.profile.image_url(size), :alt => person.name, :class => 'avatar', :title => person.name, 'data-person_id' => person.id)
+    image_tag(person.profile.image_url(size), :alt => person.name, :class => 'avatar img-responsive center-block',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.